src/components/Courses/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin: 0 auto;
+  ${({ theme }) => theme.mq.md} {
+    width: 90%;
+  } ;
+`;
+const CoursesHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin: 0 auto;
+
+  button {
+    color: #2575e6;
+    background-color: white;
+    border: 1px solid #2575e6;
+    border-radius: 10px;
+    outline: none;
+    padding: 10px 3px;
+    font-size: ${({ theme }) => theme.font.size.xxs};
+    cursor: pointer;
+  }
+  h2 {
```

Comment:
Jak już każdy element mamy jako oddzielny komponent w styled components to wrzućmy to h2 jako oddzielny również, bez sensu tak zagnieżdżać